File: cf_sdk/src/main/java/com/example/cf_sdk/logic/changebanksdk/model/transaction/TransactionFilter.java
package com.example.cf_sdk.logic.changebanksdk.model.transaction;

import com.google.gson.annotations.SerializedName;

/**
 *
 * Transaction filter.
 */

public enum TransactionFilter {

    @SerializedName("All")
    ALL,

    @SerializedName("Pending")
    PENDING,

    @SerializedName("Completed")
    COMPLETED
}

File: cf_sdk/src/main/java/com/example/cf_sdk/changebankapi/model/transaction/TransactionFilter.kt
package com.example.cf_sdk.changebankapi.model.transaction

import com.google.gson.annotations.SerializedName

/**
 *
 * Transaction filter.
 */
enum class TransactionFilter {
    @SerializedName("All")
    ALL, @SerializedName("Pending")
    PENDING, @SerializedName("Completed")
    COMPLETED
}